config/default/config_default.yaml

```diff
@@ -1,10 +1,12 @@
-agent_id: "SampleAgent"
-output_name: "sample_agent"
+agent_id: "MovieBotAgent"
+output_name: "moviebot_agent"
+# moviebot_uri is only needed to run the usage example.
+moviebot_uri: "http://127.0.0.1:5001"
```

Comment:
What about calling it `agent_uri` generally? The comment could state that by default it's assumed that the agent has a http API.

Reply:
Good point. Done.